mteb/model_meta.py

```diff
@@ -120,6 +122,7 @@ class ModelMeta(BaseModel):
     superseded_by: str | None = None
     is_cross_encoder: bool | None = None
     modalities: list[MODALITIES] = ["text"]
+    quantization: QUANTIZATION_TYPES = "float"
```

Comment:
Also this should be `None` for `API` based models. And I don't think that we want to add different annotations to voyage models, because I'm not sure if it different models or the same
```suggestion
    quantization: QUANTIZATION_TYPES | None = "float"
```

Reply:
Then we can rename it to `output_dtype`

Reply:
I don't think numpy/pytorch is needed, but output_dtype might be a better name.